app/views/searches/new.html.erb

```diff
@@ -1,21 +1,11 @@
-<% content_for(:page_title_prefix, "Search") %>
+<% content_for(:page_title_prefix, "Find information on roles") %>
 
-<h2 class="heading-xlarge">Explore work you could do</h2>
+<h2 class="heading-xlarge">Find information on roles</h2>
 
 <% if current_scrapbook.saved_occupations.any? %>
```

Comment:
`current_scrapbook.saved_occupations.any?` logic duplicated from controller. Wouldn't be better to pass this as a model to view, i.e. `showSavedRoles?`